### PR TITLE
Hotfix/adjustments

### PR DESCRIPTION
**Pull request by gloriateodoro**
- [x] Adicionar (em português e inglês) as campanhas "Meu Corpo Meu DIU" e "Pôr do Sol Sem Cerca JÁ!" nos carrosséis.
- [x] Adicionar Jerlan, Karol e Rafael às lideranças da página /about
- [x] Adicionar Alice, Laura, Paula Eduarda, Isabella, Juliana e Mariana à equipe da página /about
- [x] Atualizar fotos de Glória, Igor e Adriele.
- [x] Remover vagas de emprego já preenchidas
- [x] Adicionar "Cadê meu absorvente" ao carrossel de campanhas atuais
- [x] Remover sobrenomes da equipe
- [x] Adicionar descrição em inglês do Muvuca no carrossel de campanhas atuais

### CODE REVIEW

**Comment by bnd-operator** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **lpirola**
You can assign the PR to them by writing `/assign @lpirola` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=nossas%2Fnossas-org).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/nossas/nossas-org/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by bnd-operator** [comment]
@gloriateodoro: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr | 6cd71dc8d899937e968e1fb671444c9f16f56cb9 | [link](http://dashboard-jx.dev.nossas.org/namespaces/jx/pipelineruns/nossas-nossas-org-pr-63-pr-nrvsk) | `/test pr`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by bnd-operator** [comment]
@gloriateodoro: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr | 70964ccf9fee0b4a101e21bde907fcfe8f04ec31 | [link](http://dashboard-jx.dev.nossas.org/namespaces/jx/pipelineruns/nossas-nossas-org-pr-63-pr-87mpp) | `/test pr`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by bnd-operator** [comment]
@gloriateodoro: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr | f23f327c52680f345563c146161f7a7e714aee47 | [link](http://dashboard-jx.dev.nossas.org/namespaces/jx/pipelineruns/nossas-nossas-org-pr-63-pr-kh7q7) | `/test pr`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by bnd-operator** [comment]
:star: PR built and available in a preview **nossas-nossas-org-pr-63** [here](http://nossas-org-pr63.dev.nossas.org)

**Comment by bnd-operator** [comment]
:star: PR built and available in a preview **nossas-nossas-org-pr-63** [here](http://nossas-org-pr63.dev.nossas.org)

**Comment by bnd-operator** [comment]
:star: PR built and available in a preview **nossas-nossas-org-pr-63** [here](http://nossas-org-pr63.dev.nossas.org)

**Comment by bnd-operator** [comment]
:star: PR built and available in a preview **nossas-nossas-org-pr-63** [here](http://nossas-org-pr63.dev.nossas.org)

**Comment by bnd-operator** [comment]
:star: PR built and available in a preview **nossas-nossas-org-pr-63** [here](http://nossas-org-pr63.dev.nossas.org)

**Comment by bnd-operator** [comment]
:star: PR built and available in a preview **nossas-nossas-org-pr-63** [here](http://nossas-org-pr63.dev.nossas.org)

**Comment by bnd-operator** [comment]
:star: PR built and available in a preview **nossas-nossas-org-pr-63** [here](http://nossas-org-pr63.dev.nossas.org)

**Comment by bnd-operator** [comment]
:star: PR built and available in a preview **nossas-nossas-org-pr-63** [here](http://nossas-org-pr63.dev.nossas.org)

**Comment by bnd-operator** [comment]
:star: PR built and available in a preview **nossas-nossas-org-pr-63** [here](http://nossas-org-pr63.dev.nossas.org)

File: pages/about/_team.tsx
Comment (lpirola) [inline pages/about/_team.tsx:249]: @gloriateodoro seria bom pensar depois em padronizar o tipo de arquivo que utilizamos, talvez até adicionar um esquema otimizar automaticamente durante o build.

**Review by lpirola — APPROVED** [review]

**Comment by bnd-operator** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[lpirola](https://github.com/nossas/nossas-org/pull/63#pullrequestreview-746418841 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=nossas%2Fnossas-org).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/nossas/nossas-org/blob/main/OWNERS)~~ [lpirola]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
